src/concurrent.cr

```diff
@@ -12,15 +12,15 @@ def sleep(seconds : Number)
     raise ArgumentError.new "Sleep seconds must be positive"
   end
 
-  Fiber.sleep(seconds)
+  Crystal::Scheduler.sleep(seconds.seconds)
```

Comment:
`Crystal::Scheduler.sleep` look a bit off as it seems that the scheduler would sleep when it actually puts the current fiber to sleep. I don't have better suggestion at hand, just wanted to mention it.

Reply:
`Crystal::Scheduler.sleep` is used only used in these two methods so i think its acceptable